### PR TITLE
(Poly) Transaction Integration Test

### LINKED ISSUE #1258: Poly Transfer Integration Test
To verify functional transaction behavior of a Topl blockchain, we need an integration test which runs various transfers and ensures the correct balance change results.

Acceptance Criteria:
- A new integration test exists which tests Poly transfers
- The test should transfer poly from a genesis address to another genesis address
- The test should then transfer poly from a genesis address to a new address
- The test should then transfer poly from the new address to a "burn" address

### PR DESCRIPTION
**Pull request by SeanCheatham**
## Purpose
- Before the RPC server is updated to use the new topl-rpc library, an integration test is needed to verify that the functional behavior of the RPC server does not change
- This test simply transfers polys between a few different addresses and checks the balances
- Also: Updating to java11

## Approach
- Implemented new TransactionTest
- Copied ChainSelectionTest and BootstrapFromGenesisTest from rpc-refactor branch

## Testing
- Unit tests pass locally
- Integration test command from README passes locally

## Tickets
- closes #1258

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -274,8 +271,8 @@ lazy val node = project.in(file("node"))
   .settings(
     IntegrationTest / parallelExecution := false
   )
+  .dependsOn(common, toplRpc % "it")
```
Comment (SeanCheatham) [inline build.sbt:274]: toplRpc dependency scoped only to `node/it`, so the main Bifrost app still does not include the dependency.

File: node/src/it/scala/co/topl/it/ChainSelectionTest.scala [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.{EitherValues, Ignore, Inspectors}
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+// NOTE: This test currently fails because block difficulties diverge between nodes.  When nodes re-join, the blocks
+// can't be properly appended.
+@Ignore
```
Comment (SeanCheatham) [inline node/src/it/scala/co/topl/it/ChainSelectionTest.scala:19]: Disabling/ignoring until we've figured out the issue

File: node/src/it/scala/co/topl/it/util/DockerSupport.scala [hunk 3]
```diff
@@ -42,7 +42,15 @@ class DockerSupport(dockerClient: DockerClient)(implicit system: ActorSystem) ex
       environment.toList.map { case (key, value) => s"$key=$value" }
 
     val cmd =
-      List("--apiKeyHash", NodeRpcApi.ApiKeyHashBase58, "-c", "/opt/docker/config/testConfig.conf")
+      List(
+        "-Dcom.sun.management.jmxremote.port=9083",
```
Comment (SeanCheatham) [inline node/src/it/scala/co/topl/it/util/DockerSupport.scala:46]: Allows us to use remote JMX debugging from VisualVM

**Review by scasplte2 — APPROVED** [review]
👍

**Pull request merged**